From: Vyatta Package Maintainers <DL-vyatta-help@att.com>
Date: Mon, 21 Jan 2019 13:20:21 +0000
Subject: RoutingPlane: Use idiomatic C++11

static casting instead of C style casting
checking empty containers

Silences cpp_check warnings
---
 src/modules/ipfix/RoutingPlane.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/modules/ipfix/RoutingPlane.cpp b/src/modules/ipfix/RoutingPlane.cpp
index 3b17bc9..822fd90 100644
--- a/src/modules/ipfix/RoutingPlane.cpp
+++ b/src/modules/ipfix/RoutingPlane.cpp
@@ -986,7 +986,7 @@ RoutingPlane::get_data (routing_plane_data_t *routing_data[],
 		}
 	}
 
-	if (indexes.size() == 0) {
+	if (indexes.empty()) {
 		msg(LOG_INFO, "Routing Plane API: 100%% cache hit, no message sent");
 		ret = 0;
 		goto error;
@@ -1383,7 +1383,7 @@ RoutingPlane::run()
  */
 void *
 RoutingPlane::threadWrapper(void *routingPlane_) {
-	RoutingPlane *routingPlane = (RoutingPlane *)routingPlane_;
+	RoutingPlane *routingPlane = static_cast<RoutingPlane *>(routingPlane_);
 
 	routingPlane->run();
 
